src/user-icon/sizes.scss

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@use 'sass:math';
 @import '../style/index';
 @import '../text/mixins';
 
 @mixin iui-user-icon-size($size: $iui-icons-xl, $font-size: $iui-font-size-leading) {
   width: $size;
   height: $size;
-  --_iui-user-icon-status-size: #{math.round(math.div($size, 6))};
+
+  @if ($size == $iui-icons-xl) {
+    --_iui-user-icon-status-size: #{$iui-xs};
```

Comment:
hmm.. Not too small?
![image](https://user-images.githubusercontent.com/81580355/160384686-7d51f18a-5eb0-43b4-aee7-0faba7820545.png)

Size is 12px (for some reason),
so I guess icon should be 6px (not in our sizes :D)

Reply:
imo we shouldn't even we be showing a status icon at that size. 6px looks too big

size is 12px but it has a 2px border on all sides so it becomes 8px.

Reply:
well idk.. Now it looks super bad.. all of them except maybe green one

Reply:
| 6px | 4px |
| --- | --- |
| <img width="104" alt="image" src="https://user-images.githubusercontent.com/9084735/160386450-5214df96-d3bd-4743-b90e-b02d90b36267.png"> | 
<img width="115" alt="image" src="https://user-images.githubusercontent.com/9084735/160386591-a42dc240-0b1d-4762-ae77-15055d94fd56.png"> |

Reply:
Yeah, checkmark seems too big, but then others are at least visible :DDD
idk... idea not to show them at all at that small size is also worth thought...
@FlyersPh9 check.
I am approving from my side 6px, but removing them completely also is fine to me.